### PR TITLE
Add support for 1.20.2

### PR DESCRIPTION
**Pull request by NotMyFault**
TODO:
- [x] Update refractions
- [x] Regen
- [x] Testing

### CODE REVIEW

File: worldedit-bukkit/adapters/adapter-1_20_2/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_20_R2/PaperweightDataConverters.java [hunk 1]
```diff
@@ -0,0 +1,2801 @@
...
+
+        static {
+            DataConverterPotionId.potions[0] = "minecraft:water";
+            DataConverterPotionId.potions[1] = "minecraft:regeneration";
+            DataConverterPotionId.potions[2] = "minecraft:swiftness";
+            DataConverterPotionId.potions[3] = "minecraft:fire_resistance";
+            DataConverterPotionId.potions[4] = "minecraft:poison";
+            DataConverterPotionId.potions[5] = "minecraft:healing";
+            DataConverterPotionId.potions[6] = "minecraft:night_vision";
+            DataConverterPotionId.potions[7] = null;
```
Comment (TheMeinerLP) [inline worldedit-bukkit/adapters/adapter-1_20_2/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext/fawe/v1_20_R2/PaperweightDataConverters.java:1395]: Is the intention of be null ?

**Review by TheMeinerLP — APPROVED** [review]

**Comment by TheMeinerLP** [comment]
I have still not tested! But i will do

**Comment by MattBDev** [comment]
Looks good. If tests come back as all working I approve.

**Review by x9nico — APPROVED** [review]

**Review by PierreSchwang — APPROVED** [review]
Regen works fine so far, Entity-Copy works as well. I think that should cover the bits that are important

**Pull request merged**